src/base/torrentfilter.h

```diff
@@ -60,6 +60,7 @@ class TorrentFilter
 
     static const QString AnyCategory;
     static const QStringSet AnyHash;
+    static const QString TagFilteringDisabled;
```

Comment:
>This issue is that what these things functionally mean is "select all torrents regardless of hash / category / tag".

In other words it means "filter accepts torrents with any hash/category/tag" ("any tag" means any tag or any set of tag or no tag, there is nothing wrong) so current names are valid.